molecule/release-upgrade/group_vars/all

```diff
@@ -4,8 +4,8 @@ pulp_upgrade: true
 pulp_install_plugins:
   pulp-file:
     upgrade: true
-  pulp-rpm:
-    upgrade: true
+  # pulp-rpm:
```

Comment:
Should these still be commented out?

Reply:
it is a problem with pulp_rpm release, I need to address it in a better way, but for now I just wanted to isolate pulp_rpm and make the CI work

Reply:
It seems like this is a bug due to you accidentaly switching the upgrades for c7 and f31 to the wrong images.

Reply:
it is due to the change I did at github actions workflows, but pulp_rpm is about to be released, so I can remove the comments and go back to rpm images